contributing.md

```diff
@@ -0,0 +1,27 @@
+# How To Become A Contributor And Submit Your Own Code
+There are few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
+
+## Getting Started
+This repo uses Lerna(https://lernajs.io/) as a top level setup, which is a tool that optimizes the workflow around managing multi-package repositories with git and npm.
+
+* Make sure you have forked the desired repo(https://github.com/electrode-io/electrode.git) for developing and testing your code changes.
+
+* Make sure you have installed Lerna
+
+````
+$ npm install --global lerna
+$ lerna init
```

Comment:
don't put `lerna init` here.  that's only if you want to create a new lerna repo.